src/coordinax/_src/vectors/transform/d3.py

```diff
@@ -494,3 +495,139 @@ def represent_as(
     """
     warn("irreversible dimension change", IrreversibleDimensionChange, stacklevel=2)
     return target(r=current.r * jnp.sin(current.phi), phi=current.theta)
+
+
+# =============================================================================
+# ProlateSpheroidalPos
+
+
+# -----------------------------------------------
+# 1D
+
+
+@dispatch
+def represent_as(
+    current: ProlateSpheroidalPos, target: type[CartesianPos1D], /, **kwargs: Any
+) -> CartesianPos1D:
+    """ProlateSpheroidalPos -> CartesianPos1D.
+
+    Examples
+    --------
+    >>> import warnings
+    >>> from unxt import Quantity
+    >>> import coordinax as cx
+
+    >>> x = cx.ProlateSpheroidalPos(
+    ...     mu=Quantity(2.0, "kpc2"),
+    ...     nu=Quantity(0.5, "kpc2"),
+    ...     phi=Quantity(0.5, "rad"),
+    ...     Delta=Quantity(1.0, "kpc"),
+    ... )
+
+    >>> with warnings.catch_warnings():
+    ...     warnings.simplefilter("ignore")
+    ...     x2 = cx.represent_as(x, cx.CartesianPos1D)
+    >>> x2
+    CartesianPos1D(
+      x=Quantity[PhysicalType('length')](value=f32[], unit=Unit("kpc"))
+    )
+
+    """
+    warn("irreversible dimension change", IrreversibleDimensionChange, stacklevel=2)
+    return represent_as(represent_as(current, CartesianPos3D), target)
+
+
+@dispatch
+def represent_as(
+    current: ProlateSpheroidalPos, target: type[RadialPos], /, **kwargs: Any
+) -> RadialPos:
+    """ProlateSpheroidalPos -> RadialPos.
+
+    Examples
+    --------
+    >>> import warnings
+    >>> from unxt import Quantity
+    >>> import coordinax as cx
+
+    >>> x = cx.ProlateSpheroidalPos(
+    ...     mu=Quantity(2.0, "kpc2"),
+    ...     nu=Quantity(0.5, "kpc2"),
+    ...     phi=Quantity(0.5, "rad"),
+    ...     Delta=Quantity(1.0, "kpc"),
+    ... )
+
+    >>> with warnings.catch_warnings():
+    ...     warnings.simplefilter("ignore")
+    ...     x2 = cx.represent_as(x, cx.RadialPos)
+    >>> x2
+    RadialPos(r=Distance(value=f32[], unit=Unit("kpc")))
+
+    """
+    warn("irreversible dimension change", IrreversibleDimensionChange, stacklevel=2)
+    return represent_as(represent_as(current, CartesianPos3D), target)
+
+
+# -----------------------------------------------
+# 2D
+
+
+@dispatch
+def represent_as(
```

Comment:
This can be combined with the previous to target `AbstractPos1D`? The docstring test is good to keep